src/main/java/frc/robot/Robot.java

```diff
@@ -49,11 +50,29 @@ public void robotPeriodic() {
     CommandScheduler.getInstance().run();
   }
 
-  /** This function is called once each time the robot enters Disabled mode. */
+  /** This function is called once each time the robot enters disabled mode. */
   @Override
   public void disabledInit() {
+    if (m_autonomousCommand != null) {
+      m_autonomousCommand.cancel();
+    }
     LimelightHelpers.getLatestResults(
         IntakeLimelightConstants.INTAKE_LIMELIGHT_NAME); // It takes 2.5-3s on first run
+
+    if (m_robotContainer.matchState.realMatch) {
+      m_robotContainer.intake.pivotMotor.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.elevator.leftMotor.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.elevator.rightMotor.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.conveyor.frontMotor.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.conveyor.backMotor.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.shooter.motorA.setIdleMode(IdleMode.kCoast);
+      m_robotContainer.shooter.motorB.setIdleMode(IdleMode.kCoast);
```

Comment:
Conveyor and shooter rollers we can skip. Just pivots, elevator, and steering is fine